<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TCP Server</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>591</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="portEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ConnectBtn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>LED On/Off</string>
   </property>
   <widget class="QCheckBox" name="led21">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 21</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="led22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 22</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="led23">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 23</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Servo Motor</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Target Degree</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="degree">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="move">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Sensor</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Photoresistor Value</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sensor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
